docs/node-operators/how-to/maintain-multisig-contract.md

```diff
@@ -0,0 +1,71 @@
+```
+
+A validator with the following ranking score values will have been *demoted* to a consensus validator and will need to be *removed* to the multisig control
+```
+rankingScore.status != VALIDATOR_NODE_STATUS_TENDERMINT
+rankingScore.previousStatus == VALIDATOR_NODE_STATUS_TENDERMINT
+```
+
+Seeing any validator with the above combination of ranking-score statuses means that action must be taken to prevent rewards from being withheld by the network. If the contract is still not updated and further changes to the consensus set occur then identifying the nodes to add or remove becomes harder.
+
+## How to update the contract
+
+To update the multisig contract by adding or removing a validator, there are two steps:
+- Send an `IssueSignatures` transaction into the Vega network to prompt it to issue signatures to add/remove a signer. This signature bundle can be retrieve from a data node
+- Submit that signaure bundle to the Multisig Control contract on Ethereum to update the contract
+
+
+:::note
+If you are a node operator and supplied a `--submitter-address` when announcing your node to the network, then signature bundles will be issued automatically and the first step can be skipped.
+:::
+
+### Getting the Vega network to issue signatures
+
+To get the Vega network to generate a signature bundle that can be used to add/remove a validator from the multisig contract the following CLI command can be used
+```
+vega wallet transaction send --network NETWORK --wallet WALLET --pubkey PUBKEY TRANSACTION
+'{"issueSignatures": {"submitter": "ETH_ADDRESS", "validator_node_id": "NODE_ID", "kind": "NODE_SIGNATURE_KIND_ERC20_MULTISIG_SIGNER_ADDED"}}'
+```
+
+where `TRANSACTION` will be of the following form
+```
+'{"issueSignatures": {"submitter": "ETH_ADDRESS", "validator_node_id": "NODE_ID", "kind": "KIND"}}'
+
+NODE_ID: the identifier of the node that needs to be added or removed from the Multisig Control contract
+ETH_ADDRESS: the Ethereum address that will be used to submit the signature bundle to the Multisig Control contract
+KIND: a value that represents whether the node is to be added or remove. It can be either NODE_SIGNATURE_KIND_ERC20_MULTISIG_SIGNER_ADDED or NODE_SIGNATURE_KIND_ERC20_MULTISIG_SIGNER_REMOVED
+```
+
+:::note 
+This command *does not allow* the generation of arbitrary add and remove signatures. The Vega network is aware of which nodes need to be added or removed and the transaction will fail if submitted when it's not required.
+:::
+
+### Submitting the signature bundles to update the contract
+
+The next step involves submitting the issued signature bundle to the Multisig Control to perform the updates. This can be done using the [multisig signer tool ↗](https://validator-testnet.tools.vega.xyz/). For the transaction to be successful, the Ethereum wallet that you use to connect to the tool *must* must have the same address as the value of `ETH_ADDRESS` used in the previous step.
```

Comment:
I took this address `https://validator-testnet.tools.vega.xyz/` from the other section, I'm not sure what the mainnet one is or how to toggle the URL based on which version of the docs you're looking at?

Reply:
Let's cut references to this tool now since there's not capacity to support it

Reply:
By cut references do you mean just remove the link? Or remove the reference completely and describe the process without using the tool?

Reply:
If you'd be willing/able to describe the process without the tool, that would probably be better to work around it

Reply:
I _probably_ could if I came from the angle that the reader already knows how to use Etherscan? I don't think it'll be plausible to describe it all from zero knowledge.

Do we have any fancy-doc-things that will let me pull in the contract address from somewhere to avoid me hardcoding it?